src/ticketing/types.ts

```diff
@@ -80,7 +80,7 @@ export type FareContract = {
   minimumSecurityLevel: number;
   travelRights: TravelRight[];
   qrCode: string;
-  paymentType?: string;
```

Comment:
Arrest me if I'm wrong here, but seems like `paymentType` in practice are arrays? Haven't caused any bugs yet as far as I'm aware (it's only used one place, in `OrderDetails`, but changing it to prevent bugs in the future.